services/email_capture/javascript_validation.js

```diff
@@ -0,0 +1,38 @@
+const superagent = require('superagent')
+
+superagent
+  .get('https://mailsac.com/api/addresses/mycustomer@gmail.com/messages')
+  .set('Mailsac-Key', 'YOUR_API_KEY_HERE')
+  .then((mail) => {
+      console.log(mail.body)
+  })
+  .catch(err => {
+      console.log(err.message)
```

Comment:
`console.error` would be better, but not required. Also just dump the whole err. likely there's a `body` or `data` property with the useful information